docs/designers-developers/developers/tutorials/javascript/loading-javascript.md

```diff
@@ -27,12 +28,14 @@ We'll check the JavaScript console in your browser's Developer Tools, to see if
 
 If your code is registered and enqueued correctly, you should see a message in your console:
 
+![Console Log Message Success](../../../../../docs/designers-developers/assets/js-tutorial-console-log-success.png)
```

Comment:
With image assets I think we're cool to make them full URLs (and in fact I believe we have to right now; the handbook won't sync images and I think needs the raw URLs linked here)

Reply:
so the URL from Github, like so?

`https://raw.githubusercontent.com/WordPress/gutenberg/docs/11392/javascript/docs/designers-developers/assets/js-tutorial-console-log-error.png`

The above includes the branch, so changes after commit

Reply:
Yep, you can see how we do it here: https://raw.githubusercontent.com/WordPress/gutenberg/master/docs/designers-developers/designers/block-design.md

Reply:
Updated. 

The images won't show in PR view, since they aren't in master until after merge.